tests/ci/docker_manifests_merge.py

```diff
@@ -198,6 +272,12 @@ def main():
             if test_result != "OK":
                 status = "failure"
 
+    try:
```

Comment:
I just got, that we should make it slightly the other way around. Now the status says all images are updated, and it's wrong. 
![image](https://github.com/ClickHouse/ClickHouse/assets/3025537/7d51541e-3404-4026-9fc2-a63e824a5f4f)

I think, we instead need to preserve `updated_images = copy(changed_images)` and use them later after line 296